.github/workflows/blossom-ci.yml

```diff
@@ -42,11 +42,10 @@ jobs:
       rongou,\
       wjxiz1992,\
       GaryShen2008,\
-      pxLi,\
       NvTimLiu,\
       YanxuanLiu,\
       zhanga5,\
-      rishic3,\
+      Er1cCheng,\
```

Comment:
Strange.   This change was already merged in.   Not clear why it is showing as diff.

Reply:
I think maybe I did the merge in a wrong way so it's pulled as a diff.